https://issues.redhat.com/browse/ACM-23489 Secure clusters tech debt updates (#8239)
open-cluster-management/rhacm-docs: secure_clusters/fine_grain_rbac_cli.adoc

oc edit policy -n open-cluster-management-global-set policy-virt-clusterroles
----

.. Edit the `remediationAction` value from `inform` to `enforce`. 

> 

Reviewer: I believe there's an extra space here

+

> *Important:* Two `remediationAction` specifications are in the policy, but you only need to change the later `remediationAction`. This does not apply to the first template in the YAML file. See the following sample:

Reviewer: Disregard if I'm incorrect, but I think "This" could be clearer: 
```suggestion
*Important:* Two `remediationAction` specifications are in the policy, but you only need to change the last `remediationAction`. The specifications do not apply to the first template in the YAML file. See the following sample:
```
